scripts/decredStakeVoting.md

```diff
@@ -1,40 +1,30 @@
 # Decred In Depth: Stake Voting
 
 ### Keywords
-`unique`, `lucrative`, `investor`, `stake vote`, `no effort`, `risk-free`, `proof-of-stake`
+`unique`, `ticket holder`, `proof-of-stake`, `vote`, `governance`
 
-Decred's unique stake voting system allows users to vote with their coins to reinforce the network, choose the direction of its evolution, and earn more coins.
+Decred's unique proof-of-stake system allows stakeholders to vote with their coins to secure the network and participate in its governance.
 
-To participate in the proof-of-stake mining system, users lock a portion of their coins in exchange for a ticket. Once their ticket votes, their locked coins become spendable and they receive new minted coins as reward. 
+To take part in the proof-of-stake voting system, stakeholders purchase 'tickets' by locking a portion of their coins. 
 
-Tickets are chosen to vote randomly using arbitrary entropy generated by the blockchain.
+Purchased tickets are placed in the ticket pool where they wait to be randomly selected.
 
-A tickets votes are cast by a user chosen stake server.
+Once a ticket is drawn, the ticket holder's votes are cast by their chosen stake server; a service which provides the infrastructure to ensure votes are broadcast to the network swiftly.
 
-Stake servers stand by 24/7 to ensure votes are counted and rewards are earned, serving to eliminate losses due to computer downtime or bad Internet connections. In return, stake servers receive a small portion of the proof-of-stake rewards to help cover operational costs.
+The votes cast by ticket holders are essential to the security and governance of the Decred network. 
 
-Stake votes are integral to the integrity of the Decred network. 
+The 5 votes mined into each new block determine the validity of the previous one. If the previous block is deemed invalid, the transactions and proof-of-work reward are stripped from the network, punishing the miner for adding an invalid block to the chain. 
 
-The votes in each new block determine the validity of the previous one. If the previous block is deemed invalid, the transactions and miner portion of the block reward are stripped from the network. Stake voters, on the other hand, always receive their portion of the reward, whether they deem a block valid or not. 
+Voters, on the other hand, are always rewarded with their share of the block reward. This way voters are compensated for stopping miner attacks and preventing unwanted chain forks by rejecting blocks that break the consensus rules.
 
-This way, stakeholders are compensated for stopping miner attacks and preventing unwanted chain forks.
+Decisions regarding consensus rule changes and network updates are also made by voters through an on-chain decision-making system. Through this system stakeholders have already voted to change the algorithm responsible for setting the price of a ticket as well as to prepare the network for lightning transactions. 
```

Comment:
"network upgrades" feels slightly better

Reply:
agreed. Thanks for the feedback!